src/periphery/interfaces/ISablierMerkleBase.sol

```diff
@@ -81,4 +88,16 @@ interface ISablierMerkleBase is IAdminable {
     /// @param to The address to receive the tokens.
     /// @param amount The amount of tokens to claw back.
     function clawback(address to, uint128 amount) external;
+
+    /// @notice Withdraws the Sablier fees accrued to the provided address.
+    ///
+    /// @dev This function transfers ETH to the provided address. If the receiver is a contract, it must be able to
+    /// receive ETH.
+    ///
+    /// Requirements:
+    /// - The caller must be the Factory contract.
```

Comment:
```
/// - `msg.sender` must be the `FACTORY` contract.
```

Reply:
Do you want to replace `caller` with `msg.sender`? if so, we have been using caller to represent `msg.sender` through the natspecs.

Agree to use `FACTORY` keyword.

Reply:
> Do you want to replace `caller` with `msg.sender`? if so, we have been using caller to represent `msg.sender` through the natspecs.

yes, that's going to be consistent over core and periphery (e.g. [1](https://github.com/sablier-labs/v2-core/blob/a322ba684812fefb6fbf41cf6fd12e88904df662/src/core/interfaces/ISablierLockup.sol#L205), [2](https://github.com/sablier-labs/v2-core/blob/a322ba684812fefb6fbf41cf6fd12e88904df662/src/core/interfaces/ISablierLockup.sol#L220), [3](https://github.com/sablier-labs/v2-core/blob/a322ba684812fefb6fbf41cf6fd12e88904df662/src/core/interfaces/ISablierLockup.sol#L237))

Reply:
Sounds good. I will replace `caller` with `msg.sender` then.

Reply:
https://github.com/sablier-labs/v2-core/pull/1038/commits/155401cfafa1e2e76c64f4b23a6ec53ede5b91d9